### ISSUE TITLE
Document 'errors'

### ISSUE THREAD

**Issue by graup** [ethereum/solc-js#22]
By trial and error I found out that in case the compilation fails `compile()` returns an object with a `errors` attribute, which is a list of all compiler errors. This is really useful! Maybe it could be documented somewhere. I'm not sure here is the best place, because that already comes from the compiler I assume, but [here](https://www.npmjs.com/package/solc) was where I was looking for it.

**Comment by chevdor** [comment]
I will publish a commit in a second about that one.
btw: I totally agree!

**Comment by chevdor** [comment]
Houston, we have a problem!
I do not want to change the tests without discussing a bit further the reason for them to specifically state that the compilation *should not* produce any output:

See https://github.com/ethereum/solc-js/blob/master/test/cli.js#L24 for instance:
```
 t.test('--bin', function (st) {
    var spt = spawn(st, './solcjs --bin test/DAO/Token.sol');
    spt.stderr.empty();
    spt.stdout.empty();        <<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<
    spt.succeeds();
    spt.end();
  });
```

It especially says that the output should be empty. So my fix in 990e2f5 break those tests.

I would ask someone to confirm that this part of the test can and should be remove, then I am happy to do so.

**Comment by chevdor** [comment]
See #61
